WP_Auth0.php

```diff
@@ -372,18 +372,6 @@ public function render_auth0_login_css() {
 			return;
 		}
 
-		// If the user has a WP session, determine where they should end up and redirect.
```

Comment:
Meat of the PR here. This still was not early enough to precede any HTML output, leading to a PHP warning.